src/test/kotlin/pmeet/pmeetserver/user/resume/service/ResumeServiceUnitTest.kt

```diff
@@ -180,4 +181,34 @@ internal class ResumeServiceUnitTest : DescribeSpec({
     }
   }
 
+  describe("change status") {
+    context("이력서의 프미팅 게시 상태를 참으로 변경한다.") {
+      it("이력서의 프미팅 게시 여부가 참으로 변경된다.") {
+        runTest {
+          val capturedResumeSlot = slot<Resume>()
+          every { resumeRepository.save(capture(capturedResumeSlot)) } answers { Mono.just(capturedResumeSlot.captured) }
+
+          resumeService.changeActive(resume, true)
+
+          coVerify { resumeRepository.save(any()) }
+          assert(capturedResumeSlot.captured.isActive) { "isActive should be true" }
```

Comment:
```suggestion
          capturedResumeSlot.captured.isActive shouldBe true
```
`kotest`의 `shouldBe`를 바로 사용하지 않고 assert를 사용한 이유가 궁금합니다.